plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/GradleEnclavePlugin.kt

```diff
@@ -115,53 +119,47 @@ class GradleEnclavePlugin @Inject constructor(private val layout: ProjectLayout)
             // Make sure that the user has specified productID, print friendly error message if not
             if (!conclaveExtension.productID.isPresent) {
                 throw GradleException(
-                        "Enclave product ID not specified! " +
-                        "Please set the 'productID' property in the build configuration for your enclave.\n" +
-                        "If you're unsure what this error message means, please consult the conclave documentation.")
+                    "Enclave product ID not specified! " +
+                            "Please set the 'productID' property in the build configuration for your enclave.\n" +
+                            "If you're unsure what this error message means, please consult the conclave documentation.")
             }
             // Make sure that the user has specified revocationLevel, print friendly error message if not
             if (!conclaveExtension.revocationLevel.isPresent) {
                 throw GradleException(
-                        "Enclave revocation level not specified! " +
-                        "Please set the 'revocationLevel' property in the build configuration for your enclave.\n" +
-                        "If you're unsure what this error message means, please consult the conclave documentation.")
+                    "Enclave revocation level not specified! " +
+                            "Please set the 'revocationLevel' property in the build configuration for your enclave.\n" +
+                            "If you're unsure what this error message means, please consult the conclave documentation.")
             }
         }
 
-        target.createTask<EnclaveClassName>("enclaveClassName") { task ->
-            task.dependsOn(target.tasks.withType(JavaCompile::class.java))
-            task.inputClassPath.set(getMainSourceSet(target).runtimeClasspath)
+        val sourceFatJar: Provider<RegularFile> = if (pythonSourcePath == null) {
```

Comment:
I know that sounds silly, but why don't start to we put code like this this in functions? It is polluting